#151476592 fix landing page links

gbengaoyetade opened the pull request:
#### What does this PR do?
Makes each link on the landing page conform to its reference

#### Description of Task to be completed?
Users should be able to link from the landing page to other links provided on the landing page.

#### How should this be manually tested?
Visit the landing page of the app and click each link

#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
Connect each link on the landing page to it's destination

gbengaoyetade committed:
setup project

gbengaoyetade committed:
chore(code-base): merge develop branch
This updates the codebase locally with the one available on the git repo

gbengaoyetade committed:
feature(landing-page): fix broken links
This makes all links on the landing page refrence the correct resource
Finishes #151476592

enodi commented on app/views/includes/head.jade:
```diff
@@ -13,9 +13,11 @@ head
   link(rel="stylesheet" href='https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css')
   link(href='https://fonts.googleapis.com/icon?family=Material+Icons', rel='stylesheet', type='text/css')
+  link(rel="stylesheet" href="../css/cfh.css")
+  //- link(rel="stylesheet" href="../css/style.css")
```
remove unused code

segunolalive replied:
remove this line.

style.css is included in another line

enodi commented on public/css/style.css:
```diff
@@ -5,17 +5,17 @@
+/* header .nav-wrapper a {
 font-family: 'Satisfy'
-}
+} */
```
remove unused code

enodi commented on public/views/index.html:
```diff
@@ -233,10 +225,9 @@ <h3 class="center"><a target="_blank"  href="http://github.com/">Segun</a></h3>
       </footer>
       <!-- Latest compiled and minified Bootstrap JavaScript and Jquery-->
-      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+      <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script> -->
```
remove unused code

gbengaoyetade committed:
refactor(landing-page): remove redundant code
Some of the imports were commented out and have now been remove since they are no longer needed

gbengaoyetade committed:
Merge branch 'develop' into feature/151476592/fix-landing-page-links
